packages/devtools/src/types.ts

```diff
@@ -14,8 +14,5 @@ export interface DeployedArtifact {
 }
 
 export interface DeployedArtifacts {
-  [artifactName: string]: {
-    address: string;
-    abi: string;
-  };
+  [artifactName: string]: DeployedArtifact;
```

Comment:
If you want to add some extra niceness, you can do
```
type ArtifactName = 'Adjudicator' | 'AssetHolder';
export interface DeployedArtifacts {
  [artifactName in keyof ArtifactName]?: DeployedArtifact
}
```

This way, `DeployedArtifacts` can only have whitelisted properties, making for good autocomplete (if that's useful).

Reply:
FYI this is the same as `export type DeployedArtifacts = Partial<Record<ArtifactName, DeployedArtifact>>`